scripts/i18n/sync.js

```diff
@@ -126,7 +181,32 @@ async function syncTranslationRepo(code) {
   shell.exec(`git remote add source ${sourceRepoGitUrl}`)
   shell.exec(`git fetch source master`)
 
-  // TODO don't run the sync script if there is a current PR from the bot
+  const repository = await getRepository(owner, transRepoName)
+
+  if (repository.pullRequests.nodes.length > 0) {
+    logger.info(
+      `There are currently open sync pull requests. Please ask the language maintainers to merge the existing PR(s) in before opening another one. Exiting...`
+    )
+    process.exit(0)
+  }
+
+  logger.info(`No currently open sync pull requests.`)
+
+  let syncLabel
+  if (!repository.label) {
```

Comment:
```suggestion
  if (!repository.syncLabel) {
```

@tesseralis, if I'm understanding this correctly, I think this needs to be `syncLabel`. I'm looking at line 60. Let me know? Everything else makes sense to me!

Reply:
It should just be `label`. The `repository` here refers to the result of the query in `getRepository`, which assigns the label found using the `label` query. Although I do think it's unclear. I can rename those query variables.